<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;Arial&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1101</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1101</width>
       <height>611</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Milestone1 - Page 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>291</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(28, 50, 95);</string>
      </property>
      <property name="title">
       <string>State / City</string>
      </property>
      <widget class="QComboBox" name="stateList">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>181</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 92, 92);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
      <widget class="QListWidget" name="cityList">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>181</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 92, 92);</string>
       </property>
      </widget>
      <widget class="QListWidget" name="zipcodeList">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>190</y>
         <width>181</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 92, 92);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ZipCode</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="businessGroup">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>300</y>
        <width>531</width>
        <height>301</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(28, 50, 95);
background-color: rgb(28, 50, 95);</string>
      </property>
      <property name="title">
       <string>Businesses</string>
      </property>
      <widget class="QTableWidget" name="businessTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>511</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 92, 92);
font: 10pt &quot;Arial&quot;;</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="ZipCodeAndCategories">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>0</y>
        <width>481</width>
        <height>281</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(28, 50, 95);
background-color: rgb(28, 50, 95);</string>
      </property>
      <property name="title">
       <string>ZipCode And Categories</string>
      </property>
      <widget class="QTableWidget" name="categoryTable">
       <property name="geometry">
        <rect>
         <x>245</x>
         <y>61</y>
         <width>221</width>
         <height>201</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 92, 92);
font: 10pt &quot;Arial&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>30</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Top Categories</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ZipCode Statistics</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="zipStats1">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 92, 92);
font: 14pt &quot;Arial&quot;;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="zipStats2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 92, 92);
font: 14pt &quot;Arial&quot;;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="zipStats3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>200</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 92, 92);
font: 14pt &quot;Arial&quot;;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string># of Business</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Total Population</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Average Income</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="FilterCategories">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>181</width>
        <height>301</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(28, 50, 95);
background-color: rgb(28, 50, 95);</string>
      </property>
      <property name="title">
       <string>Filter Categories</string>
      </property>
      <widget class="QListWidget" name="categoryList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>161</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 92, 92);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Category</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>320</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 143, 0);</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>380</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 147, 0)</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="PopularBusinesses">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>0</y>
        <width>301</width>
        <height>271</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(28, 50, 95);
background-color: rgb(28, 50, 95);</string>
      </property>
      <property name="title">
       <string>Popular Businesses (in ZipCode)</string>
      </property>
      <widget class="QTableWidget" name="popularTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>281</width>
         <height>231</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 92, 92);
font: 10pt &quot;Arial&quot;;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="SuccessfulBusinesses">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>320</y>
        <width>301</width>
        <height>281</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(28, 50, 95);
background-color: rgb(28, 50, 95);</string>
      </property>
      <property name="title">
       <string>Successful Businesses (in ZipCode)</string>
      </property>
      <widget class="QTableWidget" name="successfulTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>281</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 92, 92);
font: 10pt &quot;Arial&quot;;</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="refresh">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>280</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(148, 82, 0);</string>
      </property>
      <property name="text">
       <string>Refresh Businesses</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1101</width>
       <height>611</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Milestone2 - Page 2</string>
     </attribute>
     <widget class="QGroupBox" name="searchBusiness">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>791</width>
        <height>361</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(28, 50, 95);</string>
      </property>
      <property name="title">
       <string>Search Business by Name</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Enter Business Name: </string>
       </property>
      </widget>
      <widget class="QLabel" name="citBusinessPage2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>250</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>City for Business</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="bname">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>40</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 13pt &quot;Arial&quot;;
background-color: rgb(92, 92, 92);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="bcity">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>260</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 13pt &quot;Arial&quot;;
background-color: rgb(92, 92, 92);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QListWidget" name="businesses">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>80</y>
         <width>191</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 92, 92);</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1123</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
